commcare_connect/opportunity/models.py

```diff
@@ -84,8 +84,8 @@ class Opportunity(BaseModel):
     total_budget = models.IntegerField(null=True)
     api_key = models.ForeignKey(HQApiKey, on_delete=models.DO_NOTHING, null=True)
     currency = models.CharField(max_length=3, null=True)
-    auto_approve_visits = models.BooleanField(default=False)
-    auto_approve_payments = models.BooleanField(default=False)
+    auto_approve_visits = models.BooleanField(default=True)
```

Comment:
The defaults for these columns need to be reverted back to false since the migration was removed. This will cause a new migration to be created when someone runs the makemigrations command.

Reply:
That's a good point. Though, I think we just need to add migration to change the defaults. I have added it.